src/resources/views/crud/fields/inc/translatable_icon.blade.php

```diff
@@ -1,3 +1,16 @@
-@if (method_exists($crud->model, 'isTranslatableAttribute') && ($crud->model->isTranslatableAttribute($field['name']) || (isset($field['store_in']) ? $crud->model->isTranslatableAttribute($field['store_in']) : false)) && config('backpack.crud.show_translatable_field_icon'))
+@php
+    //if field name is array we check if any of the arrayed fields is translatable
+    $translatable = false;
+    $translationsEnabled = method_exists($crud->model, 'isTranslatableAttribute');
+    if($translationsEnabled) {
```

Comment:
```suggestion
    if($crud->model->translationEnabled()) {
```

This method is available on `HasTranslatableFields` trait, through `CrudTrait`, I think it is safe to use and easy to read, what do you think?

Reply:
100% \o